### PR TITLE
chore(package): Add npm run script for webpack

### PR DESCRIPTION
**Pull request by jhermsmeier**
Make life simpler

Change-Type: patch

### CODE REVIEW

**Comment by lurch** [comment]
...so that you can do `npm webpack` in addition to `make webpack` ?

**Comment by lurch** [comment]
From one of the Travis build logs...
```
cp package.json dist/Etcher-cli-1.4.4+ae5ae6a-linux-x64-app

mkdir dist/Etcher-cli-1.4.4+ae5ae6a-linux-x64

cd dist/Etcher-cli-1.4.4+ae5ae6a-linux-x64-app && pkg --output ../../dist/Etcher-cli-1.4.4+ae5ae6a-linux-x64/etcher -t node6-linux-x64 lib/cli/etcher.js

> pkg@4.3.0

> Fetching base Node.js binaries to PKG_CACHE_PATH

[1G  fetched-v6.11.5-linux-x64    [] 0%[0K[1G  fetched-v6.11.5-linux-x64    [] 14%[0K[1G  fetched-v6.11.5-linux-x64    [] 33%[0K[1G  fetched-v6.11.5-linux-x64    [] 34%[0K[1G  fetched-v6.11.5-linux-x64    [] 35%[0K[1G  fetched-v6.11.5-linux-x64    [] 36%[0K[1G  fetched-v6.11.5-linux-x64    [] 37%[0K

> Error! ESOCKETTIMEDOUT

> Asset not found by direct link:

  {"tag":"v2.5","name":"uploaded-v2.5-node-v6.11.5-linux-x64"}

> Error! API rate limit exceeded for 35.192.85.2. (But here's the good news: Authenticated requests get a higher rate limit. Check out the documentation for more details.)

make: *** [dist/Etcher-cli-1.4.4+ae5ae6a-linux-x64] Error 2


travis_time:end:0462531e:start=1526659363512916646,finish=1526659696427516899,duration=332914600253
[0K
[31;1mThe command "./scripts/ci/build-installers.sh -o $HOST_OS -r $TARGET_ARCH" exited with 2.[0m
```

Perhaps it's worth investigating if `PKG_CACHE_PATH` can be persisted between TravisCI runs, to prevent these rate-limit errors?

File: webpack.config.js [hunk 1]
```diff
@@ -64,16 +64,19 @@ module.exports = {
       {
         test: /\.jsx?$/,
         include: [ path.resolve(__dirname, 'lib/gui') ],
+        exclude: /node_modules/,
```
Comment (lurch) [inline webpack.config.js]: I know nothing about webpack, but does this change relate to #2195 ?

**Comment by jhermsmeier** [comment]
@lurch `$HOME/.pkg-cache` is already cached on CI – there's a new version of that binary available in this case from what I can tell. Also it'll still have to hit the API to figure out whether the cache is stale :/

**Pull request merged**